### PR TITLE
[T4A3][F10-B2] Shi Xiyue 

### PR DESCRIPTION
**Pull request by shixiyue**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -6,26 +6,22 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact{
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Address.java:14]: why public?

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Contact.java:3]: header comments

**Comment by KulaniT** [comment]
@shixiyue  Overall nice work. Practice to give the lowest possible access modifier and to add comments

File: src/seedu/addressbook/data/person/Contact.java (hunk 2 above)
Comment (shixiyue) [reply src/seedu/addressbook/data/person/Contact.java:3]: Ok! I will add header comments.

**Pull request closed without merging**